Makefile

```diff
@@ -195,19 +200,20 @@ bundle: $(KUSTOMIZE) $(OPERATOR_SDK) generate
 		 	$(BUNDLE_METADATA_OPTS)
 	sed -e 's|<IMG_OBSERVABILITY_OPERATOR>|$(OPERATOR_IMG)|g' \
 		-i bundle/manifests/observability-operator.clusterserviceversion.yaml
-	$(OPERATOR_SDK) bundle validate ./bundle \
-		--select-optional name=operatorhub \
-		--optional-values=k8s-version=1.21 \
-		--select-optional suite=operatorframework
-	git diff --quiet -I'^    createdAt: ' bundle && git checkout bundle || true
+	$(OPERATOR_SDK) bundle validate ./bundle
 
 .PHONY: bundle-image
 bundle-image: bundle ## Build the bundle image.
-	$(CONTAINER_RUNTIME) build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+	$(CONTAINER_RUNTIME) build --no-cache -f bundle.Dockerfile --platform linux/amd64 -t $(BUNDLE_IMG)-amd64 .
+	$(CONTAINER_RUNTIME) build --no-cache -f bundle.Dockerfile --platform linux/ppc64le -t $(BUNDLE_IMG)-ppc64le .
+	$(CONTAINER_RUNTIME) push $(BUNDLE_IMG)-amd64
+	$(CONTAINER_RUNTIME) push $(BUNDLE_IMG)-ppc64le
```

Comment:
I'd prefer the image push stays in the push target. Is this needed to build the manifest?

Reply:
Yes if we're using Podman, we can't build and push in one command for Podman multi-arch builds like Docker